projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -807,7 +806,7 @@ function add_paywall( $the_content ) {
 	}
 
 	require_once JETPACK__PLUGIN_DIR . 'extensions/blocks/premium-content/_inc/subscription-service/include.php';
-	$token_service              = is_wpcom() ? new WPCOM_Token_Subscription_Service() : new Jetpack_Token_Subscription_Service();
+	$token_service              = is_wpcom() ? new Jetpack_Token_Subscription_Service() : new Jetpack_Token_Subscription_Service();
```

Comment:
this if doesn't make much sense?  😆

I think we should use something like: https://github.com/Automattic/jetpack/blob/1b98ac60b97a2b3e31f42200bc89f34b656160fa/projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/include.php#L43 whenever we need a `Jetpack_Token_Subscription_Service`